pymongoexplain/__main__.py

```diff
@@ -48,6 +49,18 @@ def new_func(self: Collection, *args, **kwargs):
     setattr(Collection, old_func_name, make_func(old_func, old_func_name))
 
 if __name__ == '__main__':
-    for file in sys.argv[1:]:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "input_script", action="store", nargs=1, help="The script that you "
+                                                      "wish to run explain on.")
+    parser.add_argument(
+        "arguments", metavar="script_arguments", help="add arguments to "
+                                                      "explained script",
+                        action="store", nargs="?")
+
+    args = parser.parse_args()
+    for file in args.input_script:
```

Comment:
Why is this in a for loop if it can only contain 1 value?

Reply:
Not sure, fixed.